mscore/shortcut.cpp

```diff
@@ -701,9 +701,9 @@ Shortcut Shortcut::_sc[] = {
       {
          MsWidget::SCORE_TAB,
          STATE_NORMAL | STATE_NOTE_ENTRY,
-         "add-sforzato",
```

Comment:
I wonder whether this change should be done, it'd invalidate existing shortcuts.
Also requires changes to the 3 shortcuts*.xml we distribute